### ISSUE TITLE
sys.database_scoped_configurations describes wrong columns

### ISSUE THREAD

**Issue by chojrak11** [MicrosoftDocs/sql-docs#1266]
https://docs.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-database-scoped-configurations-transact-sql?view=sql-server-2017

In SQL Server 2016 I'm able to fetch the following columns from this DMV with `SELECT *`:
`configuration_id`, `name`, `value`, `value_for_secondary`
In SQL Server 2017 and SQL Server 2019 CTP2.1:
`configuration_id`, `name`, `value`, `value_for_secondary`, `is_value_default`

which is not what's documented. Please fix it.

**Comment by jzabroski** [comment]
Confirmed. On a brand new install on a AWS RDS SQL Server 2017 Standard Edition:
```sql
SELECT
	dsc.*
FROM sys.database_scoped_configurations dsc
```
returns:
```
configuration_id name                          value value_for_secondary is_value_default
---------------- ----------------------------- ----- ------------------- ----------------
1                MAXDOP                        0     NULL                1
2                LEGACY_CARDINALITY_ESTIMATION 0     NULL                1
3                PARAMETER_SNIFFING            1     NULL                1
4                QUERY_OPTIMIZER_HOTFIXES      0     NULL                1
6                IDENTITY_CACHE                1     NULL                1

(5 rows affected)
```

**Comment by MightyPen** [comment]
@chojrak11 Thanks to you opening this issue, it has been fixed now, with PR #1281.  So we are Closing this Issue as fixed.
Thank you.

CC: @VanMSFT
